android/arecord

```diff
@@ -1,22 +1,36 @@
 #!/bin/bash
+
 LOCATION=$(dirname "$0")
 source "$LOCATION"/../common_tools
-trap 'ctrlc $@' 1 2 3 6 15
 
```

Comment:
add "check dependency scrcpy" to handle cases where the user does not have it installed yet